src/aui-button/js/aui-button-core.js

```diff
@@ -50,11 +51,23 @@ ButtonExt.ATTRS = {
      * CSS class to be automatically added to the `boundingBox`.
      *
      * @attribute cssClass
-     * @type String
+     * @type {String}
      */
     cssClass: {
         validator: isString,
-        value: CLASS_NAMES.BUTTON_DEFAULT
+        value: ''
+    },
+
+    /**
+     * Defines if the button will discard the default button classes.
+     *
+     * @attribute discardDefaultButtonCssClasses
+     * @default false
+     * @type {Boolean}
+     */
+    discardDefaultButtonCssClasses: {
```

Comment:
You don't add attribute change cycle (afterChange/uiSet....), if you sure about this decision please add the writeOnce to this attribute.